cartridge/failover.lua

```diff
@@ -730,6 +804,57 @@ local function cfg(clusterwide_config)
             end,
         })
         vars.failover_fiber:name('cartridge.stateful-failover')
+
+    elseif failover_cfg.mode == 'raft' then
+        box.cfg{
+            replication_connect_quorum = 0,
+            -- The instance is set to candidate, so it may become leader itself
+            -- as well as vote for other instances.
+            --
+            -- Alternative: set one of the three instances to `voter`, so that it
+            -- never becomes a leader but still votes for one of its peers and helps
+            -- it reach election quorum (2 in our case).
+            election_mode = os.getenv('TARANTOOL_ELECTION_MODE') or 'candidate',
+            -- Quorum for both synchronous transactions and
+            -- leader election votes.
+            replication_synchro_quorum = failover_cfg.replication_synchro_quorum or 2,
```

Comment:
Since 2.6.2  the option can be expression string
`replication_synchro_quorum = "N/2 + 1"`